mountainview/src/mvoverview2widget.cpp

```diff
@@ -1292,7 +1299,7 @@ void MVOverview2WidgetPrivate::open_clips()
     X->setMscmdServerUrl(m_mscmdserver_url);
     X->setProperty("widget_type", "clips");
     X->setProperty("ks", int_list_to_string_list(ks));
-    q->connect(X, SIGNAL(currentEventChanged()), q, SLOT(slot_clips_view_current_event_changed()));
+    q->connect(X, SIGNAL(currentEventChanged()), q, SLOT(slot_clips_widget_current_event_changed()));
```

Comment:
Since `connect()` is a static function, it looks nicer to write:
`QObject::connect(X, SIGNAL(currentEventChanged()), q, SLOT(slot_clips_widget_current_event_changed()));`

By the way, naming slots `slot_clips_widget_current_event_changed` is a really bad habbit. You should describe what the function does and not when it is executed.